web/ui/react-app/src/pages/targets/StateIndicator.module.css

```diff
@@ -0,0 +1,4 @@
+.message {
+  padding: 0 4px 0 4px !important;
```

Comment:
I'm curious why !important ?

Reply:
Otherwise they won't be applied because the default styles get applied instead. I couldn't figure out a better way of doing this. 😭

Reply:
yes, you can add it like a style attr ```style={{ padding:....}}```

Reply:
ah yes that's true

Reply:
i wish there were a way to specify that my stylesheet takes precedence without having to put the styles directly in the js code

Reply:
You can try to make stronger selector than the default one. If the default one is 
```.foo.bar { default styles here }```
you selector should be ```.your-css-class .foo.bar { your styles here }```

Reply:
used the inline styles attr here